runtime/runtime-domain-service/java/build.gradle

```diff
@@ -15,37 +15,21 @@
  *
  */
 
-plugins {
-    id 'base'
-    id 'jacoco-report-aggregation'
-}
-
-repositories {
-    mavenCentral()
-}
-
-
-reporting {
-    reports {
-        testCodeCoverageReport(JacocoCoverageReport) {
-            testType = TestSuiteType.UNIT_TEST
-        }
-    }
-}
-
-tasks.named('check') {
-    dependsOn tasks.named('testCodeCoverageReport', JacocoReport)
-}
+apply from: "$projectDir/../../../common-gradle-scripts/java.gradle"
 
 tasks.register('java_build') {
     group 'Java build'
-    println "build java"
+    doLast {
+        println "build java"
+    }
 }
 
 tasks.named('build').configure {
     dependsOn java_build
 }
```

Comment:
Do we need those tasks? `build` task is anyway included with the java plugin.

Reply:
Fix https://github.com/wso2/apk/pull/168/commits/052c9414f83b73a6c109f63e19fd982335b98c5f